pkgs/servers/matrix-synapse/matrix-appservice-irc/update.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env nix-shell
+#! nix-shell -i bash -p nodePackages.node2nix nodejs-12_x curl jq nix-update
+
+set -euo pipefail
+# cd to the folder containing this script
+cd $(dirname $0)
+
+CURRENT_VERSION=$(jq -r '.[0]."matrix-appservice-irc" | split("#") | .[1]' package.json)
```

Comment:
```suggestion
CURRENT_VERSION="$(nix eval --raw '(with import ./. {}; matrix-appservice-irc.version)')"
```

Reply:
`error: anonymous function at /home/piegames/Documents/git/nix/nixpkgs2/pkgs/servers/matrix-synapse/matrix-appservice-irc/default.nix:1:1 called without required argument 'pkgs', at (string):1:7`

One would need to import the `nixpkgs` and then `callPackage`, at which point one could also simply import the nixpkgs and do `pkgs.matrix-appservice-irc.version`?

Reply:
```
❯ pwd
/home/hexa/git/nixos/master
❯ nix eval --raw '(with import ./. {}; matrix-appservice-irc.version)'
0.26.1%
```

Reply:
Ah, yes. The script wants to be executed in the folder itself, so using `import ../../..` or something should do.